TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationComponents.tsx

```diff
@@ -22,8 +23,9 @@ import { TIMEOUT_S } from './CalibrationScreen';
 interface CalibrationInstructionsProps {
     progress: number;
     containerStyle: CSSProperties;
+    top?: boolean;
```

Comment:
We could make this isTop (and then pass in false for the bottom one) meaning we can remove the containerStyle props as this is just a different div style depending on top or bottom

Or could make CalibrationInstructionsTop and CalibrationInstructionsBottom which just calls CalibrationInstructions with the correct isTop boolean??

Reply:
Updated name, left the rest as discussed.